### DISCUSSION TITLE
切换Python版本后导致终端打不开

### DISCUSSION THREAD

**Discussion by caibingcheng** [caibingcheng/comment#295] [Announcements]
https://www.bbing.com.cn/202208/terminal-python-version/

**Comment by penndu** [comment]
记得检查一下apt可以正常使用否！

**Comment by caibingcheng** [comment]
> @penndu
> 记得检查一下apt可以正常使用否！

apt可用, 不过software updater这个软件用不了(里面用到了async作为变量名, 但是3.7将async升级为关键字和3.6不兼容, 导致语法错误, 不过幸好没多大影响, 用apt update/upgrade也行)

**Comment by penndu** [comment]
> @caibingcheng
> > @penndu
> > 记得检查一下apt可以正常使用否！
> 
> apt可用, 不过software updater这个软件用不了(里面用到了async作为变量名, 但是3.7将async升级为关键字和3.6不兼容, 导致语法错误, 不过幸好没多大影响, 用apt update/upgrade也行)

对桌面用户还是有影响！

**Comment by ryan4yin** [comment]
改系统环境坑多

**Comment by caibingcheng** [comment]
> @ryan4yin
> 改系统环境坑多

是的, 严重的系统都进不去了~~
